client/network/src/protocol.rs

```diff
@@ -754,6 +758,11 @@ impl<B: BlockT, H: ExHashT> Protocol<B, H> {
 				message_queue: None,
 				justification,
 			};
+			// Stop if we don't have requested block body
+			if get_body && block_data.body.is_none() {
+				trace!(target: "sync", "Missing data for block request. BODY: {}", block_data.body.is_some());
```

Comment:
```suggestion
				trace!(target: "sync", "Missing data for block request.");
```

This will only be printed when `body.is_none()`.